### PR TITLE
feat: Healthz api check appId support

### PR DESCRIPTION
**Pull request by alberthuang24**
related: https://github.com/dapr/components-contrib/issues/2489

Signed-off-by: AlbertHuang <r.albert.huang@gmail.com>

# Description

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Unit tests passing
* [x] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by alberthuang24** [comment]
@ItalyPaleAle I successfully turned on "Allow edits and access to secrets by maintainers"

File: pkg/http/api_test.go [hunk 1]
```diff
@@ -5019,6 +5021,20 @@ func TestV1HealthzEndpoint(t *testing.T) {
 		assert.Equal(t, 204, resp.StatusCode)
 	})
 
+	t.Run("Healthz - 500 No AppId Match", func(t *testing.T) {
```
Comment (daixiang0) [inline pkg/http/api_test.go:5024]: Need empty ID case.

Comment (alberthuang24) [reply pkg/http/api_test.go:5024]: This case has been available before

https://github.com/dapr/dapr/pull/6279/files#diff-3e12a060b83c3a6da274fd9de083fdec28ed255a49a5c347530eade7acee2e2bL5007-L5018

**Comment by alberthuang24** [comment]
Hi, is there anything else that needs to be done with this PR so that it can be merged?

**Review by daixiang0 — APPROVED** [review]

**Comment by daixiang0** [comment]
@alberthuang24 please fix conflicts.

**Review by ItalyPaleAle — APPROVED** [review]

**Comment by ItalyPaleAle** [comment]
This was merged with #6642

**Pull request closed without merging**